android-lib/src/main/java/com/kinvey/android/AndroidUserStore.kt

```diff
@@ -43,16 +43,15 @@ internal class AndroidUserStore private constructor(context: Context?) : ClientU
         editor?.commit()
     }
 
-    /** {@inheritDoc}  */
-    override fun setUser(userID: String) {
-        this.userID = userID
-        persistData()
-    }
-
-    /** {@inheritDoc}  */
-    override fun getUser(): String? {
-        return userID
-    }
+    override var user: String? = null
+        get() {
+            return userID
+        }
+        set(value) {
+            field = value
```

Comment:
Yes, logic is correct, I combined getter/setter for `userID` field that was in `Java` code.
Also improved it, removed redundant code `field = value`, because `field` value is never read in this `user` property.